### DISCUSSION TITLE
Cannot clone git repo for postal

### DISCUSSION THREAD

**Discussion by dcgleason** [postalserver/postal#2036] [Help with using Postal]
Hi, 

I am trying to clone the git repo with this command:

git clone https://postalserver.io/start/install /opt/postal/install

I have git. Also, this command returns the following error: 'fatal: could not create leading directories of '/opt/postal/install': Permission denied'

Also tried gh repo clone https://postalserver.io/start/install /opt/postal/install

and got -- 

HTTP 404: 404 Not Found (https://postalserver.io/api/graphql)

Any idea what I should do? I am following the prerequisite steps found here: https://docs.postalserver.io/install/prerequisites#git--installation-helper-repository

Thank you,
Dan

**Comment by willpower232** [comment]
I believe you will need to be `root` to do the initial clone.
